src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -882,6 +886,21 @@ private void ExecuteFindReferences(ITextBuffer subjectBuffer, IContentType conte
                 lastHandler: executeNextCommandTarget);
         }
 
+        protected virtual void SyncClassView(SnapshotPoint caretPosition)
+        {
+        }
+
+        void ExecuteSyncClassView(ITextBuffer subjectBuffer, IContentType contentType, Action executeNextCommandTarget)
```

Comment:
Actually, trying to get back from a command handler to the package to find the object browser is proving ridiculously hard, and requiring a bunch of forwarding methods that I don't like.

Any ideas?